pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManager.java

```diff
@@ -489,6 +489,9 @@ protected boolean consumeLoop()
     }
 
     if (_numRowsErrored > 0) {
+      _serverMetrics.addMeteredGlobalValue(ServerMeter.REALTIME_CONSUMPTION_EXCEPTIONS, _numRowsErrored);
```

Comment:
I don't think this is the correct place to update the meter. This meter is used to track the consume exception, `_numRowsErrored` is used to track the invalid rows. IMO we should update the meter under the catch on line 422 and 425

Reply:
good idea. will do so